### PR TITLE
release.sh: initial script

### PR DESCRIPTION
**Pull request by FFY00**
Signed-off-by: Filipe Laíns <lains@archlinux.org>

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+echo
+
+read -p 'Are you sure you want to proceed? (y/n) ' -n 1 -r < /dev/tty
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+    echo 'Release aborted.'
+    exit 1
+fi
+
+read -p 'Are you *relly* sure you want to proceed? (y/n) ' -n 1 -r < /dev/tty
```
Comment (Klap-in) [inline release.sh]: relly=really

Comment (FFY00) [reply release.sh]: Ah, thanks for catching that!

**Comment by pfps** [comment]
This is the first part of a release, right?  Then you have to get into github and create the release.  I think there is a way to POST the release information, but doing it using the regular github UI is a good idea as far as I am concerned.

**Comment by FFY00** [comment]
I should also document the release proccess. There are some requirements, mainly HEAD should point to a release commit.

**Comment by FFY00** [comment]
That should be working now.

**Pull request merged**